processor/.dockerignore

```diff
@@ -2,3 +2,4 @@
 *Dockerfile*
 *docker-compose*
 node_modules
+dist
```

Comment:
What's in dist?

Reply:
Compiled js from ts files

Reply:
Isn't that exactly what *should* go in the docker container?

Reply:
.dockerignore is used when we copy files from the host to the container, the `dist` dir gets generated inside the container itself when compiling. Copying the host `dist` dir causes permission problems (and will cause problems if the container is running on windows too).

Reply:
Ok, thanks for explaining.